.goreleaser.yml

```diff
@@ -50,14 +50,66 @@ archives:
     <<: *archive_defaults
     wrap_in_directory: false
     format: zip
+aurs:
+  -
+    homepage: "https://github.com/fastly/cli"
+    description: "A CLI for interacting with the Fastly platform"
+    maintainers:
+      - 'oss@fastly.com'
+    license: "Apache license 2.0"
+    skip_upload: auto
+
+    # The SSH private key that should be used to commit to the Git repository.
+    # This can either be a path or the key contents.
+    #
+    # WARNING: do not expose your private key in the config file!
+    private_key: '{{ .Env.AUR_KEY }}'
+
+    # The AUR Git URL for this package.
+    # Defaults to empty.
+    git_url: 'ssh://aur@aur.archlinux.org/cli-bin.git'
```

Comment:
Not sure what this should be set to.

Reply:
I think we may want this to be `fastly-bin`. Because I think folks will search AUR for "fastly". And would probably expect to see `fastly`, `fastly-bin` or `fastly-git` is the pattern if you provide all three.

`fastly-bin` would be if you just served a binary. I think `fastly-git` would be if you were going to clone and build from source. And I can't remember what the plain `fastly` would be.

Reply:
I always reference https://github.com/Jguer/yay-PKGBUILD because `yay` is currently the most common tool for interacting with the AUR.

Reply:
So to be clear, I think you are correct using the `-bin` suffix here.